frontend/src/components/ExpensePresentation.js

```diff
@@ -1,28 +1,19 @@
 import React from "react";
-import { Table } from "react-bootstrap";
+import "../styles/Expenses.css";
```

Comment:
Should be ExpensePresentation.css

Reply:
Yes, all in separate files.

Reply:
Resolved in 98ae7b35ef0c274798064fc2b47af099f5e6cfa2